apps/betterangels-backend/clients/migrations/0008_clientprofile_california_id.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 5.1.3 on 2024-11-12 04:56
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("clients", "0007_clientprofile_important_notes"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="clientprofile",
+            name="california_id",
+            field=models.CharField(
+                blank=True,
+                max_length=8,
+                null=True,
+                unique=True,
+                validators=[
+                    django.core.validators.RegexValidator(
+                        message="California ID must be 1 letter followed by 7 numbers", regex="^[A-Z]\\d{7}$"
```

Comment:
would you mind noting this to the ticket and share? 
it's business logic and just wanna make sure it's documented

Reply:
good call. [confirmed](https://betterangels.atlassian.net/browse/DEV-907?focusedCommentId=10765)